packages/reactstrap-validation-select/resource-components.js

```diff
@@ -15,34 +16,35 @@ export const PatientOption = props => {
 
   const option = (
     <Fragment>
-      <strong>
-        {lastName}, {firstName} {middleName ? `${middleName}.` : ''}
-      </strong>
-      &nbsp;&nbsp;&nbsp;{payerName} {memberId || subscriberMemberId}
+      <Row>
```

Comment:
I believe adding `noGutters` prop to the `Row` tag will make sure there is no scroll bar on the component unless the content actually exceeds the max length of the container.

Reply:
a687af1d4bf939e6d11fd367786c085e8024d417